Library/Homebrew/keg_relocate.rb

```diff
@@ -67,9 +67,13 @@ def text_files
     path.find do |pn|
       next if pn.symlink? || pn.directory?
       next if Metafiles::EXTENSIONS.include? pn.extname
-      if Utils.popen_read("/usr/bin/file", "--brief", pn).include?("text") ||
-         pn.text_executable?
-        text_files << pn
+      # File has known issues with reading files on other locales.
+      # http://bugs.gw.com/view.php?id=292
+      with_custom_locale("C") do
```

Comment:
```
vim (Known problematic), before PR:

brew bottle vim  25.07s user 12.28s system 101% cpu 36.910 total
brew bottle vim  24.97s user 11.92s system 100% cpu 36.651 total
brew bottle vim  24.89s user 11.90s system 101% cpu 36.350 total

vim (Known problematic), after PR:

brew bottle vim  24.86s user 11.82s system 100% cpu 36.363 total
brew bottle vim  24.56s user 11.81s system 102% cpu 35.538 total
brew bottle vim  24.59s user 11.65s system 103% cpu 35.085 total


openssl (Known fine), before PR:

brew bottle openssl  17.76s user 7.23s system 102% cpu 24.395 total
brew bottle openssl  17.53s user 7.00s system 103% cpu 23.736 total
brew bottle openssl  17.47s user 6.77s system 103% cpu 23.402 total


openssl (Known fine), after PR:

brew bottle openssl  18.17s user 7.14s system 102% cpu 24.722 total
brew bottle openssl  17.68s user 6.97s system 102% cpu 24.099 total
brew bottle openssl  17.61s user 6.89s system 102% cpu 23.856 total


Martin's proposed rejig:
brew bottle openssl  17.47s user 6.79s system 103% cpu 23.374 total
brew bottle openssl  17.56s user 6.97s system 103% cpu 23.723 total
brew bottle openssl  17.46s user 6.96s system 103% cpu 23.644 total
brew bottle vim  23.81s user 11.49s system 105% cpu 33.582 total
brew bottle vim  23.72s user 11.52s system 106% cpu 33.034 total
brew bottle vim  24.24s user 11.70s system 104% cpu 34.358 total
```

Not a giant different either way, but agree there's a minor performance gain here so will adopt the proposed rejiggle.